lib/Quantization/Base/Base.cpp

```diff
@@ -149,6 +150,18 @@ TensorQuantizationParams chooseQuantizationParams(float min, float max) {
   min = std::min(min, 0.f);
   max = std::max(max, 0.f);
 
+  if (schema == quantization::Schema::Symmetric) {
```

Comment:
can you add some tests to make sure that you'll get the 0 offsets

Reply:
Good point! Forgot to do that after testing the loader.

Reply:
@rdzhabarov, I've added a test case for the quantization schema if you want to take a look.